notebooks/nessie-iceberg-demo-nba.ipynb

```diff
@@ -0,0 +1,629 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Nessie Spark SQL Demo with NBA Dataset\n",
+    "============================\n",
+    "This demo showcases how to use Nessie Python API along with Spark3 from Iceberg\n",
+    "\n",
+    "Initialize Pyspark\n",
+    "----------------------------------------------\n",
+    "To get started, we will first have to do a few setup steps that give us everything we need\n",
+    "to get started with Nessie. In case you're interested in the detailed setup steps for Spark, you can check out the [docs](https://projectnessie.org/tools/spark/)\n",
```

Comment:
nit: `.` at the end of the sentence. Same for the the next sentence

Reply:
done